### PR TITLE
[enhancement](bitmap)support bitmap type for non-key column in duplicate table

### PR DESCRIPTION
**Pull request by bobhan1**
## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by dataroaring — DISMISSED** [review]
LGTM

**Comment by bobhan1** [comment]
run buildall

**Comment by bobhan1** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/analysis/CreateTableStmt.java [hunk 1]
```diff
@@ -523,12 +523,7 @@ public void analyze(Analyzer analyzer) throws UserException, AnalysisException {
             }
 
             if (columnDef.getType().isObjectStored()) {
-                if (columnDef.getType().isBitmapType()) {
-                    if (keysDesc.getKeysType() == KeysType.DUP_KEYS) {
-                        throw new AnalysisException("column:" + columnDef.getName()
-                                + " must be used in AGG_KEYS or UNIQUE_KEYS.");
-                    }
-                } else {
+                if (!columnDef.getType().isBitmapType()) {
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/CreateTableStmt.java:526]: please fix nereids CreateTableCommand too

File: fe/fe-core/src/main/java/org/apache/doris/analysis/CreateTableStmt.java [hunk 2]
```diff
@@ -523,12 +523,7 @@ public void analyze(Analyzer analyzer) throws UserException, AnalysisException {
             }
 
             if (columnDef.getType().isObjectStored()) {
-                if (columnDef.getType().isBitmapType()) {
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/CreateTableStmt.java:526]: do we check bitmap could not be key?

**Comment by bobhan1** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by yiguolei — APPROVED** [review]
lgtm

**Review by zhannngchen — APPROVED** [review]
LGTM

**Pull request merged**